lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -597,14 +619,16 @@ class EmitterBase {
 } // end anonymous namespace
 
 void EmitterBase::emitTextWithSubstitutions(
-    StringRef string, Operation *op,
-    std::function<void(Value)> operandEmitter) {
+    StringRef string, Operation *op, std::function<void(Value)> operandEmitter,
+    llvm::Optional<SmallVector<Operation *, 8>> symOps,
```

Comment:
Plz pass this as non-optional `ArrayRef<Operation*>`

Reply:
Also, why not pass in the ArrayRef of symbol references, and have this code resolve them, rather than do it in the caller?

Reply:
Passing the operations, since we need to resolve the symbols, and only the caller has access to the symbol table cache, otherwise we have to pass the symbol table also as an argument.